spec/services/bulk_upload/sales/year2023/row_parser_spec.rb

```diff
@@ -959,14 +959,24 @@ def purch_id_field
     end
 
     describe "#field_67 - 70" do # buyers organisations
-      context "when all nil" do
-        let(:attributes) { setup_section_params.merge(field_67: nil, field_68: nil, field_69: nil, field_70: nil) }
-
-        it "returns correct errors" do
-          expect(parser.errors[:field_67]).to be_present
-          expect(parser.errors[:field_68]).to be_present
-          expect(parser.errors[:field_69]).to be_present
-          expect(parser.errors[:field_70]).to be_present
+      let(:empty_organisation_params) { setup_section_params.merge(field_67: nil, field_68: nil, field_69: nil, field_70: nil) }
+
+      context "when all empty" do
+        let(:attributes) { empty_organisation_params }
+
+        it "sets pregblank field" do
+          expect(parser.log.pregblank).to be(1)
+        end
+      end
+
+      %i[field_67 field_68 field_69 field_70].each do |field_number|
```

Comment:
This feels a little clunky - very open to suggestions on how to make it cleaner.